Helper/Order.php

```diff
@@ -738,6 +738,15 @@ public function processExistingOrder($quote, $transaction)
     {
         // check if the order has been created in the meanwhile
         if ($order = $this->getExistingOrder($quote->getReservedOrderId())) {
+
+            if($order->isCanceled()) {
```

Comment:
Should we check below any other status?
As example `on hold `.

Does exist such case?